SmartDeviceLink/public/SDLSetGlobalProperties.h

```diff
@@ -69,7 +69,24 @@ NS_ASSUME_NONNULL_BEGIN
  @param menuLayout The layout of the top-level main menu
  @return The SetGlobalProperties RPC
  */
-- (instancetype)initWithHelpText:(nullable NSString *)helpText timeoutText:(nullable NSString *)timeoutText vrHelpTitle:(nullable NSString *)vrHelpTitle vrHelp:(nullable NSArray<SDLVRHelpItem *> *)vrHelp menuTitle:(nullable NSString *)menuTitle menuIcon:(nullable SDLImage *)menuIcon keyboardProperties:(nullable SDLKeyboardProperties *)keyboardProperties menuLayout:(nullable SDLMenuLayout)menuLayout;
+- (instancetype)initWithHelpText:(nullable NSString *)helpText timeoutText:(nullable NSString *)timeoutText vrHelpTitle:(nullable NSString *)vrHelpTitle vrHelp:(nullable NSArray<SDLVRHelpItem *> *)vrHelp menuTitle:(nullable NSString *)menuTitle menuIcon:(nullable SDLImage *)menuIcon keyboardProperties:(nullable SDLKeyboardProperties *)keyboardProperties menuLayout:(nullable SDLMenuLayout)menuLayout __deprecated_msg("Use initWithHelpText:timeoutText:vrHelpTitle:vrHelp:menuTitle:menuIcon:keyboardProperties:userLocation:menuLayout: instead");
+
+/**
+ Initialize SetGlobalProperties with all possible items
+
+ @param helpPrompt A string that will be turned into TTS chunks for the help prompt
+ @param timeoutPrompt A string that will be turned into TTS chunks for the timeout prompt
+ @param vrHelpTitle The title of the help interface prompt
+ @param vrHelp The items of the help interface prompt
+ @param menuTitle The title of the menu button
+ @param menuIcon The icon on the menu button
+ @param keyboardProperties The properties of a keyboard prompt
+ @param userLocation userLocation
+ @param menuLayout The layout of the top-level main menu
+ @return The SetGlobalProperties RPC
+ */
+
```

Comment:
```suggestion
/**
 * Convenience init for setting all possible global properties
 * 
 * @param userLocation - userLocation
 * @param helpPrompt - helpPrompt
 * @param timeoutPrompt - timeoutPrompt
 * @param vrHelpTitle - vrHelpTitle
 * @param vrHelp - vrHelp
 * @param menuTitle - menuTitle
 * @param menuIcon - menuIcon
 * @param keyboardProperties - keyboardProperties
 * @param menuLayout - menuLayout
 * @return A SDLSetGlobalProperties object
 */
```

Reply:
I just copied the generated docs for the init since the order of the parameters in old doc do not match the order for the new init. I added a custom description since it is not created by the RPC spec generator.